src/schema/objects/metadata.yaml

```
@@ -140,6 +140,17 @@ AnatomicalLandmarkCoordinates__mri:
       type: number
     minItems: 3
     maxItems: 3
+ScreenAOIDefinition:
+  name: ScreenAOIDefinition
+  display_name: Screen Area of Interest Definition
+  description: |
+    Define, what shape are the Screen AOIs and what coordinates are used.
+     ["square", ["x_start", "x_stop", "y_start", "y_stop"]]`
+    Other options:
```

```suggestion
     ["square", ["x_start", "x_stop", "y_start", "y_stop"]]`

    Other options:
```

An empty line will make sure these are rendered on separate lines.

---

Good